spec/controllers/errors_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe ErrorsController, :type => :controller do
+
+  describe "GET file_not_found" do
+    it "returns http success" do
+      get :file_not_found
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET unprocessable" do
+    it "returns http success" do
+      get :unprocessable
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET internal_server_error" do
+    it "returns http success" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.